sign_language_datasets/datasets/dgs_corpus/dgs_corpus.py

```diff
@@ -208,7 +209,19 @@ def _generate_examples(self, data):
             if self._builder_config.include_video:
                 videos = {t: str(datum["video_" + t]) if ("video_" + t) in datum else "" for t in ["a", "b", "c"]}
 
-                features["fps"] = self._builder_config.fps if self._builder_config.fps is not None else default_fps
+                fps_expected_by_loader = self._builder_config.fps if self._builder_config.fps is not None else default_fps
+
+                for video_path in videos:
+                    if video_path == "":
+                        continue
+
+                    # make sure that the video fps is as expected
+                    actual_video_fps = cv2.VideoCapture(video_path).get(cv2.CAP_PROP_FPS)
```

Comment:
`cv2.VideoCapture(video_path)` creates an open capture to the file, and needs to be `release`d or `close`d I believe.
```py
cap = cv2.VideoCapture(video_path)
actual_video_fps = cap.get(cv2.CAP_PROP_FPS)
cap.release()
```